content/documents/dev/building_haiku_ubuntu_linux_step_step.html

```diff
@@ -1,92 +1,82 @@
 +++
 type = "article"
 title = "Building Haiku on Ubuntu Linux, Step by Step"
-date = "2007-07-21T04:11:17.000Z"
-tags = ["obsolete"]
+date = "2019-03-05T11:19:23.0530Z"
 +++
+# Architecture
 
+To check whether your target architecture is support visit [Port Status](https://www.haiku-os.org/guides/building/port_status).
 
-    <div class="field field-type-text field-field-body">
-    <div class="field-items">
-            <div class="field-item odd">
-                    <div class="box-info">
-<strong>2010-January: This document is now obsolete. We are in the process of consolidating and re-organizing the website documentation. For now please refer to the <a href="/guides">in-progress website documentation</a>.</strong>
-</div>
+# Pre-requisite Software
```

Comment:
For the section headers you can decide to either use title casing ("Pre-requisite Software") or sentence casing ("Pre-requisite software"). Whatever you choose, stick with it for all headers in the document.